packages/frontend/index.d.ts

```diff
@@ -108,6 +108,7 @@ interface AuthorType {
 interface Block {
     id: string;
     elements: CAPIElement[];
+    createdOn?: number;
```

Comment:
Can this be a `Date` type instead?

Reply:
Not really. 

By convention (to avoid any type of datetime manipulation on dotcom rendering) we send to dotcom rendering preformatted datetime strings. This is, for instance, the reason we send `"createdOnDisplay": "09.58 BST"` in the JSON object. 

The introduction of `createdOn` fits perfectly the requirement for the `amp-live-list` AMP component which recommend from the documentation:

> Timestamp used for sorting entries. Higher timestamps will be inserted before older entries. We recommend using Unix time (the number of seconds that have elapsed since Thursday, 1 January 1970)

There was no need to ever pass a `Date` type. It's not needed and might lead to the temptation to datetime manipulation where by conventions we are not doing any :)

Reply:
Ok, cool, thanks!